AUTHORS.md

```diff
@@ -0,0 +1,9 @@
+# The libfulltext authors
```

Comment:
Maybe link the `AUTHORS.md` from the `README.md` ?

Reply:
agree